src/descriptors/power_system_structs.json

```diff
@@ -2333,9 +2333,10 @@
         "docstring": "Parameters of a AVR that returns a fixed voltage to the rotor winding",
         "fields": [
             {
-                "name": "Emf",
-                "comment": "Fixed voltage to the rotor winding",
+                "name": "V_ref",
```

Comment:
I am not sure we should move to use V_ref for the fix AVR since it is meant to represent a fixed emf even if the voltage at the terminals can't be kept. I think this is a different model that can't guarantee to keep the voltage at a PV bus at the set level.

Reply:
Yes. I agree with that. Will revert the change, but modify the name to Vf, that is the standard name used in LITS.

Reply:
Addressed via abdb0fa